### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by arturo-seijas**
Applicable spec: <link>

### Overview

Update the readme to match the changes in https://github.com/canonical/is-charms-template-repo

### Checklist

- [x] The [charm style guide](https://juju.is/docs/sdk/styleguide) was applied
- [x] The [contributing guide](https://github.com/canonical/is-charms-contributing-guide) was applied
- [x] The changes are compliant with [ISD054 - Managing Charm Complexity](https://discourse.charmhub.io/t/specification-isd014-managing-charm-complexity/11619)
- [x] The documentation is generated using `src-docs`
- [x] The documentation for charmhub is updated
- [x] The PR is tagged with appropriate label (`urgent`, `trivial`, `complex`)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,35 +11,18 @@ is an XML-based open-standard for transferring identity data between two parties
...
-* [Get support](https://discourse.charmhub.io/)
-* [Join our online chat](https://chat.charmhub.io/charmhub/channels/charm-dev)
-* [Contribute](https://charmhub.io/indico/docs/how-to-contribute)
-Thinking about using the Indico Operator for your next project? [Get in touch](https://chat.charmhub.io/charmhub/channels/charm-dev)!
-
-## Contributing to this documentation
-
-Documentation is an important part of this project, and we take the same open-source approach to the documentation as the code. As such, we welcome community contributions, suggestions and constructive feedback on our documentation. Our documentation is hosted on the [Charmhub forum](https://charmhub.io/saml-integrator/docs) to enable easy collaboration. Please use the "Help us improve this documentation" links on each documentation page to either directly change something you see that's wrong, ask a question, or make a suggestion about a potential change via the comments section.
+## Learn more
+* [Read more](https://charmhub.io/saml-integrator) <!--Link to the charm's official documentation-->
```
Comment (nrobinaubertin) [inline README.md:21]: Should we remove the comments ?

Comment (arturo-seijas) [reply README.md]: Oops. Good point

File: README.md [hunk 2]
```diff
@@ -11,35 +11,18 @@ is an XML-based open-standard for transferring identity data between two parties
...
-offerings and virtual machines or bare metal.
+of configuration for all the requirers using the same SAML entity.
 
-As such, the charm makes it easy to manage and propagate SAML configuration, while
-giving the freedom to deploy on the substrate of their choice.
+## Get started
 
-For DevOps or SRE teams this charm will make operating any charm leveraging SAML
-authentication simple and straightforward through Juju's clean interface.
+You can follow the tutorial [here](https://charmhub.io/saml-integrator/docs/tutorial-getting-started).
```
Comment (erinecon) [inline README.md:19]: Are there any basic operations (e.g. actions) or integrations worth mentioning for this charm?

File: README.md [hunk 3]
```diff
@@ -11,35 +11,18 @@ is an XML-based open-standard for transferring identity data between two parties
 an identity provider (IdP) and a service provider (SP).
 
 This charm simplifies configuration of SAML SPs by providing a single point
-of configuration for all the requirers using the same SAML entity. It can be
-deployed on many different Kubernetes platforms, from [MicroK8s](https://microk8s.io)
-to [Charmed Kubernetes](https://ubuntu.com/kubernetes), public cloud Kubernetes
-offerings and virtual machines or bare metal.
+of configuration for all the requirers using the same SAML entity.
```
Comment (erinecon) [inline README.md:14]: Nitpick: Personally I think this paragraph was fine as it was before, but I don't have a strong opinion about it.

File: README.md [hunk 4]
```diff
@@ -11,35 +11,18 @@ is an XML-based open-standard for transferring identity data between two parties
...
-* [Join our online chat](https://chat.charmhub.io/charmhub/channels/charm-dev)
-* [Contribute](https://charmhub.io/indico/docs/how-to-contribute)
-Thinking about using the Indico Operator for your next project? [Get in touch](https://chat.charmhub.io/charmhub/channels/charm-dev)!
-
-## Contributing to this documentation
-
-Documentation is an important part of this project, and we take the same open-source approach to the documentation as the code. As such, we welcome community contributions, suggestions and constructive feedback on our documentation. Our documentation is hosted on the [Charmhub forum](https://charmhub.io/saml-integrator/docs) to enable easy collaboration. Please use the "Help us improve this documentation" links on each documentation page to either directly change something you see that's wrong, ask a question, or make a suggestion about a potential change via the comments section.
+## Learn more
+* [Read more](https://charmhub.io/saml-integrator)
+* [Developer documentation](https://charmhub.io/saml-integrator/docs)
```
Comment (erinecon) [inline README.md:22]: I think the developer documentation should be different than our docs. I don't really know of a good external resource for developing SAML (If it even exists).

What do you think? If there's no ideal developer docs for the charm, then I think it's best to remove the line entirely.

File: README.md [hunk 5]
```diff
@@ -11,35 +11,18 @@ is an XML-based open-standard for transferring identity data between two parties
...
-If there's a particular area of documentation that you'd like to see that's missing, please [file a bug](https://github.com/canonical/saml-integrator-operator/issues).
-
----
-
-For further details,
-[see the charm's detailed documentation](https://charmhub.io/saml-integrator/docs).
+## Project and community
+* [Issues](https://github.com/canonical/saml-integrator-operator/issues)
+* [Contributing](https://charmhub.io/saml-integrator/docs/how-to-contribute)
+* [Matrix](https://chat.charmhub.io/charmhub/channels/charm-dev)
```
Comment (erinecon) [inline README.md:28]: Should this link point to Matrix instead of Mattermost?

File: README.md [hunk 6]
```diff
@@ -11,35 +11,18 @@ is an XML-based open-standard for transferring identity data between two parties
...
-* [Contribute](https://charmhub.io/indico/docs/how-to-contribute)
-Thinking about using the Indico Operator for your next project? [Get in touch](https://chat.charmhub.io/charmhub/channels/charm-dev)!
-
-## Contributing to this documentation
-
-Documentation is an important part of this project, and we take the same open-source approach to the documentation as the code. As such, we welcome community contributions, suggestions and constructive feedback on our documentation. Our documentation is hosted on the [Charmhub forum](https://charmhub.io/saml-integrator/docs) to enable easy collaboration. Please use the "Help us improve this documentation" links on each documentation page to either directly change something you see that's wrong, ask a question, or make a suggestion about a potential change via the comments section.
+## Learn more
+* [Read more](https://charmhub.io/saml-integrator)
+* [Developer documentation](https://charmhub.io/saml-integrator/docs)
+* [Troubleshooting](https://chat.charmhub.io/charmhub/channels/charm-dev)
```
[the remark this answers is not in the document]
Comment (arturo-seijas) [reply README.md:23]: I should probably remove that

File: README.md (hunk 4 above)
Comment (arturo-seijas) [reply README.md:22]: I'm changing that line for a link to the SAML specification

File: README.md (hunk 2 above)
Comment (arturo-seijas) [reply README.md:19]: No actions. And no integrations as a requirer

File: README.md (hunk 3 above)
Comment (arturo-seijas) [reply README.md:14]: I changed it to be better aligned with https://github.com/canonical/is-charms-template-repo and the changes in the Indico operator README. Let me know if you'd prefer to keep it as it was

**Review by erinecon — APPROVED** [review]
LGTM, thanks for updating!

**Review by amandahla — APPROVED** [review]

**Pull request merged**